.gitignore

```diff
@@ -1,2 +1,4 @@
 setup.sh
-.vscode
\ No newline at end of file
+.vscode
+.nodeos
+nohup.out
```

Comment:
Those can be added to the `.git/info/exclude` folder, since this is related to your local setup and not from the whole project. Related docs: http://git-scm.com/docs/gitignore